lm_eval/tasks/tasks.md

```diff
@@ -0,0 +1,100 @@
+
+# Tasks
+
+ A list of supported tasks can be viewed with `lm-eval --tasks list`
+
+| Task Family | Description |
+|-------------|-------------|
+| [aclue](aclue/README.md) | Tasks focusing on ancient Chinese language understanding and cultural aspects. |
+| [aexams](aexams/README.md) | Tasks related to various academic exams covering a range of subjects. |
```

Comment:
```suggestion
| [aexams](aexams/README.md) | Tasks in Arabic related to various academic exams covering a range of subjects. |
```

would it be possible to explicitly note when a task is multilingual or non-English? And state either the language if there is only one, or state "multilingual, with a number of language variants"?